packages/dashboard/e2e/tests/constants.ts

```diff
@@ -21,3 +21,6 @@ export const REMOVE_TREND_CURSOR = 'Delete trend cursor';
 export const TREND_CURSOR_TABLE_HEADER =
   '.base-chart-legend-tc-header-container';
 export const TREND_CURSOR_TABLE_CELL = '.trend-cursor-value';
+
+export const MAX_VALUE_TABLE_HEADER = '.base-chart-legend-max-header-container';
```

Comment:
could these be `data-testid` instead of classes? that way we guarantee they stick around since classes can change names